english/bite-the-marshmallow/lamping/lamping.md

```diff
@@ -0,0 +1,2 @@
+As you begin to scroll through your feed on IG and using your 
```

Comment:
This file is in the right place! You can fix the link to this page by changing it to `lamping/lamping.md` rather than just `lamping.md`. That should fix both this "not reachable" issue and the broken link issue.